Automattic/wp-calypso
# Add blue color scheme from wp-admin

One part of #47036

#### Changes proposed in this Pull Request

* Add blue color scheme

#### Testing instructions

* Visit /me/account
* Choose blue and save

To see the wp-admin color scheme, create an atomic site, then visit /wp-admin/profile.php, choose color scheme and save.

Calypso:
![2020-11-06_13-46](https://user-images.githubusercontent.com/937354/98408225-6fbe5700-2036-11eb-99f0-9b7f5ebbee3c.png)

Wp-Admin:
![2020-11-06_13-46_1](https://user-images.githubusercontent.com/937354/98408245-764cce80-2036-11eb-9712-b4754abf2fe2.png)
